browser-test/src/admin_settings.test.ts

```diff
@@ -0,0 +1,15 @@
+import {BASE_URL} from './support/config'
+import {createTestContext, loginAsAdmin, validateScreenshot} from './support'
+
+describe('Managing system-wide settings', () => {
+  const ctx = createTestContext()
+
+  it('Displays the settings page', async () => {
```

Comment:
It would not. It's a pain to get the request object to the layout that renders the nav, so we've usually skipped supporting feature flag overrides for the nav bar portion of the feature, which is the only thing this flag is guarding.

Reply:
Sounds good!